.github/workflows/ci-build.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.19
+          go-version: 1.20.10
```

Comment:
talked about this in person but just for documentation- instead of writing 1.20 here (because it's resolving to 1.2) or having to say specifically 1.20.10 (which means we don't get automatic upgrades when new versions come out), we should have `go-version: '1.20'`

Reply:
Thanks @reginapizza 
updated